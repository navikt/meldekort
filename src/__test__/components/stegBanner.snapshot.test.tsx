import * as React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import ProviderWrapper from "../testSetup/providerWrapper";
import StegBanner from "../../app/components/stegBanner/stegBanner";

Enzyme.configure({ adapter: new Adapter() });

it("StegBanner", () => {
  const tree = shallow(
    <ProviderWrapper>
      <StegBanner />
    </ProviderWrapper>
  );
  expect(tree).toMatchSnapshot();
});
