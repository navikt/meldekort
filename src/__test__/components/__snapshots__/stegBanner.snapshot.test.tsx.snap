// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StegBanner 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Connect(IntlProvider)
    defaultLocale="nb"
    locale="nb"
  >
    <Connect(StegBanner) />
  </Connect(IntlProvider)>
</Provider>
`;
